routes/store.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+// Admin (Service) Authentication Routes...
```

Comment:
So these should be store now, not admin?

Are they using the same controllers or all pathed under store vs admin namespaceing?

Reply:
yep